app/services/followup_service.rb

```diff
@@ -16,9 +16,7 @@ def self.create_followup(case_contact, creator, note)
     followup
   end
 
-  private_class_method
-
-  def self.send_notification(followup, creator)
+  private_class_method def self.send_notification(followup, creator)
```

Comment:
this  fixed a warning: private_class_method is ignored unless called with a symbol, like:
`private_class_method :send_notification`
or used this way.